code/solr/tasks/Solr_Reindex.php

```diff
@@ -65,7 +70,7 @@ protected function doReindex($request)
         // this is for when index names do not match the class name (this can be done by overloading getIndexName() on
         // indexes
         if ($index && !ClassInfo::exists($index)) {
-            foreach(ClassInfo::subclassesFor('SolrIndex') as $solrIndexClass) {
+            foreach(ClassInfo::subclassesFor('SilverStripe\FullTextSearch\Solr\SolrIndex') as $solrIndexClass) {
```

Comment:
I think you need to create something like this to mysite folder which is somewhat this method is for.

`class MyIndex extends SolrIndex {
    function init() {
	    $this->addClass('SilverStripe\CMS\Model\SiteTree');
        //$this->addAllFulltextFields();
	    $this->addFulltextField('Title');
	    $this->addFulltextField('Content');
    }
}`